src/Microsoft.ML.StaticPipe/TransformsStatic.cs

```diff
@@ -1550,18 +1550,18 @@ public static Vector<float> FeaturizeText(this Scalar<string> input, Scalar<stri
         }
     }
 
-    public static class RffStaticExtenensions
+    public static class RandomFourierKernelMappingStaticExtenensions
```

Comment:
>RandomFourierKernelMappingStaticExtenensions [](start = 24, length = 44)

we decided to go with acronyms

Reply:
So this would be RFKMStaticExtensions?

---
In reply to: [264836908](https://github.com/dotnet/machinelearning/pull/2888#discussion_r264836908) [](ancestors = 264836908)

Reply:
Not anymore. Its acronyms will be `AKM` which is not a usual term. At least, it's my first time of seeing that.

---
In reply to: [264869094](https://github.com/dotnet/machinelearning/pull/2888#discussion_r264869094) [](ancestors = 264869094,264836908)